Add toggles and figures

xm0onh opened the pull request:
This PR adds a toggle component to sections requiring this functionality. Furthermore, it adds a couple of figures/diagrams from Subnomicon.

dariolina commented on docs/consensus/proof_of_archival_storage.md:
```diff
@@ -119,9 +118,10 @@ This extra data is added to the end of the [SCALE-encoded](https://docs.substrat
+<center>Figure 3: Piece Building Process corresponds to steps 5-12 of Archiving.</center>
+
 
 Figure 3: Piece Building Process corresponds to steps 5-12 of Archiving. --->
```
```suggestion

```
Repeats the caption above

dariolina commented on docs/consensus/proof_of_space.md:
```diff
@@ -72,20 +73,18 @@ Computes the 7 PoS tables.
-<!-- 
-TODO: replace diagram
-![Steps 3.b and 3.c in Generate. Numbers and matching function $M$ are solely illustrative. Since the `entry.y` is sorted, Chia does this sequentially on portions of the table as their tables don’t fit in memory.  Notice that 235 and 380 appear in multiple matches, while some numbers don’t have a match.](Dilithium%20PoS%20Specification%2008339021db504ce9aa0be6cc2f52a949/Subspace_v2_Master_-_Plott ...
+![Steps 3.b and 3.c in Generate. Numbers and matching function $M$ are solely illustrative. Since the `entry.y` is sorted, Chia does this sequentially on portions of the table as their tables don’t fit in memory.  Notice that 235 and 380 appear in multiple matches, while some numbers don’t have a match.](/img/Proof_of_space_plotting.png)
```
```suggestion
![Steps 4.ii-iv in generate function definition. Numbers and matching function $M$ are solely illustrative. Since the `entry.y` is sorted, Chia does this sequentially on portions of the table as their tables don’t fit in memory.  Notice that 235 and 380 appear in multiple matches, while some numbers don’t have a match.](/img/Proof_of_space_plotting.png)
```

dariolina commented on docs/consensus/proof_of_time.md:
```diff
@@ -62,9 +64,8 @@ Takes a `seed` and number of iterations `pot_iterations` and repeatedly evaluate
-<!-- 
-TODO: replace diagram
-![Screenshot 2023-06-01 at 13.48.06.png](Dilithium%20PoT%20Specification%204e17a6d5b03a4abea864ba2d0b97970e/Screenshot_2023-06-01_at_13.48.06.png) -->
+![PoT Evaluator time in iteration.png](/img/PoT_Evaluator_time_in_iter.png)
```
This should be this one https://github.com/subspace/subnomicon/blob/main/static/img/Proof_of_Time-light.svg and the corresponding dark one

dariolina commented on docs/consensus/proof_of_time.md:
```diff
@@ -268,38 +272,33 @@ Depending on how far `block_number` is from the `target` we define a few thresho
+
+
+<Collapsible title="Note">
+    We may want to accept the fork after this probabilistic verification passes, but eventually and asynchronously verify it fully. See discussion on time [here](https://forum.subspace.network/t/pot-verification-during-genesis-sync/1606). We don’t know if it’s [implementable](https://www.notion.so/Dilithium-PoT-Specification-4e17a6d5b03a4abea864ba2d0b97970e?pvs=21)
```
```suggestion
    We may want to accept the fork after this probabilistic verification passes, but eventually and asynchronously verify it fully. See discussion on time [here](https://forum.subspace.network/t/pot-verification-during-genesis-sync/1606). 
```
Not for public eyes :)

dariolina commented on docs/decex/bundles_blocks.md:
```diff
@@ -52,10 +54,11 @@ Domain block follows the standard [Substrate block format](https://github.com/pa
-        
-        Substrate header also contains a *`digest`* field, but it is usually unused for the domain block. It was used to feed some consensus chain data into the domain block before but will likely not be used going forward if it causes a challenge for fraud proofs.
-         -->
+    <br />
```
What's this for?

dariolina commented on docs/decex/interfaces.md:
```diff
@@ -28,6 +28,9 @@ last_update:
 - `StakeWithdrawalLockingPeriod`: The number of consensus chain blocks after a staking withdrawal has been submitted and cleared before it will be transferred back to owner balances. Currently, 14400 domain blocks *(Value TBD)*
 - `BlockTreePruningDepth`: The confirmation depth at which domain blocks are pruned from the `BlockTree`*, de-facto challenge period.* Currently, 14400 domain blocks.
 - `BundleLongevity`: How long the bundle is consider as not stale, defined in the number of consensus blocks. Currently, 5 consensus blocks.
+:::warning `MaxFraudProofSize`
+    The maximum size of a fraud proof, as enforced by the fraud proof storage metering scheme. This ensures that fraud proof size is not unbounded. The default is 1 MiB.
+:::
```
```suggestion

```
This is not necessary, it's commented out because it doesn't exist and we're not sure it will.

dariolina reviewed (commented):
Thanks for going through the toggles! Left a few nits:

xm0onh commented:
Thanks!
I made the changes.

dariolina commented:
Also, could you go through the files and adjust the `last_update` in the header accordingly? Even better if you could check whether there is a way to do it automatically.

xm0onh commented:
It's done.
I added a bash script to track staged MD files and update the "latest-update" automatically. But we should use pre-commit package. I updated the README file and explained how we can set up and use it.

dariolina commented:
That's great to have an automatic tool! Thanks!
